packages/ndla-ui/src/locale/messages-en.ts

```diff
@@ -1031,6 +1031,7 @@ const messages = {
         responses_plural: "responses",
         topicContent: "Write your topic here",
         fetchMore: "Fetch more topics",
+        locked: "Lock for new posts",
```

Comment:
I nodebb står det "Lock topic". Er det like greit? Eller var tanken å ha ett innslag som skal dekke både aksjonen å låse samt det at innlegget er låst?

Reply:
Jeg syns det bare funka litt dårlig på norsk, men jeg er åpen for det altså.

Reply:
Er kanskje ikke viktig at direkteoversettelsen er lik heller. Oppdaterte.